.github/workflows/run-pr-tests.yaml

```
@@ -42,28 +42,111 @@ jobs:
     uses: ./.github/workflows/warp-nc-tests.yaml
 
   build-noobaa-image:
+    name: Build Noobaa Image
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 4

(indentation)